arch/arm/core/aarch64/reset.S

```diff
@@ -54,12 +54,12 @@ GTEXT(__start)
 SECTION_SUBSEC_FUNC(TEXT,_reset_section,__start)
 
 	/* Setup vector table */
-	adr	x19, _vector_table
+	adr	x2, _vector_table
```

Comment:
'x2' can be clobbered, we need to use a callee saved reg as now we provide a hook for platform to do el3 specific init if needed. ` z_arch_el3_plat_init`

Reply:
missed the call to `z_arch_el3_plat_init`. ACK. Fix coming.